tests/gemini/test_gemini_autolog.py

```diff
@@ -0,0 +1,66 @@
+from unittest.mock import patch
+
+import google.generativeai as genai
+import pytest
+
+import mlflow
+from mlflow.entities.span import SpanType
+
+from tests.tracing.helper import get_traces
+
+CANDIDATES = [{"content": {"parts": [{"text": "test answer"}], "role": "model"}}]
+
+USER_METADATA = {
+    "prompt_token_count": 6,
+    "candidates_token_count": 6,
+    "total_token_count": 6,
+    "cached_content_token_count": 0,
+}
+
+
+def test_enable_disable_autolog():
+    genai.GenerativeModel.generate_content = lambda self, contents: {
+        "candidates": CANDIDATES,
+        "usage_metadata": USER_METADATA,
+    }
+    mlflow.gemini.autolog()
+    model = genai.GenerativeModel("gemini-1.5-flash")
+    model.generate_content("test content")
+
+    traces = get_traces()
+    assert len(traces) == 1
+    assert traces[0].info.status == "OK"
+    assert traces[0].info.execution_time_ms > 0
+    assert len(traces[0].data.spans) == 1
+    span = traces[0].data.spans[0]
+    assert span.name == "GenerativeModel"
+    assert span.span_type == SpanType.LLM
+    assert span.inputs["contents"] == "test content"
```

Comment:
```suggestion
    assert span.inputs == {"contents": "test content"}
```

nit: to ensure `self` is removed from the input span